src/Compilers/CSharp/Portable/CSharpResources.Designer.cs

```diff
@@ -20,7 +20,7 @@ namespace Microsoft.CodeAnalysis.CSharp {
     // class via a tool like ResGen or Visual Studio.
     // To add or remove a member, edit your .ResX file then rerun ResGen
     // with the /str option, or rebuild your VS project.
-    [global::System.CodeDom.Compiler.GeneratedCodeAttribute("System.Resources.Tools.StronglyTypedResourceBuilder", "15.0.0.0")]
+    [global::System.CodeDom.Compiler.GeneratedCodeAttribute("System.Resources.Tools.StronglyTypedResourceBuilder", "4.0.0.0")]
```

Comment:
@jaredpar VS automatic Change. I usually keep them since it will just annoy other people editing the same file (and they would have to undo it as well). Should I undo here?

---
In reply to: [132530259](https://github.com/dotnet/roslyn/pull/21006#discussion_r132530259) [](ancestors = 132530259)